src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -71,6 +71,17 @@ namespace winrt::TerminalApp::implementation
         _RecalculateAndApplyTabColor();
     }
 
+    void TerminalTab::_SetToolTip(const winrt::hstring& tabTitle)
+    {
+        auto toolTip = WUX::Controls::ToolTip{};
+        auto newTabRun = WUX::Documents::Run();
+        newTabRun.Text(tabTitle);
+        auto textBlock = WUX::Controls::TextBlock{};
+        textBlock.Inlines().Append(newTabRun);
+        toolTip.Content(textBlock);

```

Comment:
we might be able to get away with `tooltip.Content(winrt::box_value(tabTitle))` and avoid the Run and TextBlock completely. Does that work?